### PR TITLE
7841 changed PDF PCA filename

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

**Comment by domdinicola** [comment]
[ch7841]

File: src/etools/applications/partners/views/v1.py [hunk 1]
```diff
@@ -39,18 +43,18 @@ def get_context_data(self, **kwargs):
             return {"error": "Cannot find document with given query parameter lang={}".format(lang)}
 
         try:
-            agreement = Agreement.objects.get(id=agr_id)
+            self.agreement = Agreement.objects.get(id=agr_id)
```
Comment (ntrncic) [inline src/etools/applications/partners/views/v1.py]: why use self here?

Comment (domdinicola) [reply src/etools/applications/partners/views/v1.py]: I set self.agreement cause I can use it in get_pdf_filename method

**Comment by ntrncic** [comment]
@domdinicola why is is for next sprint?

File: src/etools/applications/partners/views/v1.py [hunk 2]
```diff
@@ -28,6 +28,10 @@ class PCAPDFView(LoginRequiredMixin, PDFTemplateView):
         "ifrc_french": "partners/pca/ifrc_french_pdf.html"
     }
 
+    def get_pdf_filename(self):
+        return '{0.reference_number}-{0.partner}-{1.profile.country.schema_name}.pdf'.format(
```
Comment (robertavram) [inline src/etools/applications/partners/views/v1.py]: use agreement_number here instead of reference_number

**Pull request merged**